packages/core-data/package.json

```diff
@@ -32,6 +32,7 @@
 	"dependencies": {
 		"@babel/runtime": "^7.16.0",
 		"@wordpress/api-fetch": "file:../api-fetch",
+		"@wordpress/block-editor": "file:../block-editor",
```

Comment:
This feels like a code smell. Should `core-data` really depend on `block-editor`?

I understand that this was added here to support the changes in `entity-provider.js`, but I was assuming those changes were there for the prototype and not as a deliberate design choice.

It's not easy, but we should discuss our options. One of the most obvious is adding hooks for entities, e.g. `entity.onInput` and `entity.onChange`. What else can we do?

Reply:
This change also caught my eye. Maybe we should move the `useEntityBlockEditor` hook to the `editor` package; it seems more suitable, IMO.

Reply:
@youknowriad What would you recommend?

Reply:
Now tracked in https://github.com/WordPress/gutenberg/issues/64604.